docs/products/influxdb.rst

```diff
@@ -0,0 +1,27 @@
+Aiven for Aiven for InfluxDB®
+=============================
+
+Aiven for Apache InfluxDB® is a fully managed time series database, deployable in the cloud of your choice. 

```

Comment:
Incorrect product name, not sure how this happened? Influx isn't an apache project.